src/client/src/app/site/[site]/view-availability/daily-appointments/daily-appointments-page.tsx

```diff
@@ -112,12 +100,35 @@ export const DailyAppointmentsPage = ({
 
     return { headers, rows };
   };
+
   const appointmentsTableData = mapTableData();
 
   return (
     <>
       {appointmentsTableData && <Table {...appointmentsTableData}></Table>}
-      <Pagination previous={buildPreviousPage()} next={buildNextPage()} />
+
+      {/* TODO: Implement the pagination component and in the first screenshot of https://nhsd-jira.digital.nhs.uk/browse/APPT-367 and replace these links
```

Comment:
Not sure this should be a to do item. The initial version of this page will use the simplified paging components. A future version of this page might look different but that should be as per user feedback from the beta rather than assuming how it should be done

Reply:
Good point, when I did this I thought the more complex pagination was definitely queued up for a later ticket. I'll extend the existing pagination component to accept onClick links as well as href ones